### PR TITLE
enhancing the helps

### PR DESCRIPTION
**Pull request by Vilax**
- A main help explaining the algorithm has been added as well as parameter helps

- We should think about removing erase gold beads. It only removed those gold beads used for aligning the tilt series.

- The apply alignment option was removed, check please, this is a main modification

- We also should think about merging both nova protocols.

### CODE REVIEW

File: novactf/protocols/protocol_tomoReconstruction.py [hunk 1]
```diff
@@ -382,7 +402,7 @@ def _validate(self):
         validateMsgs = []
         ts = self.getInputTs()
 
-        if self.applyAlignment and not ts.getFirstItem().getFirstItem().hasTransform():
+        if not ts.getFirstItem().getFirstItem().hasTransform():
```
Comment (azazellochg) [inline novactf/protocols/protocol_tomoReconstruction.py:405]: if you are removing applyAlignment param, the protocol will fail for unaligned tilt-series.

**Review by azazellochg — COMMENTED** [review]
Thanks @Vilax !
1) I agree with removing erase gold beads. I spent a lot of time on that and could not figure out the correct beads positions.
2) merging two protocols is not feasible because you don't know how many defocus steps you get per each TS, so it's not possible to know number of protocol steps for the second protocol.

File: novactf/protocols/protocol_tomoReconstruction.py (hunk 1 above)
Comment (pconesa) [reply novactf/protocols/protocol_tomoReconstruction.py:405]: Shoudn't we trust the hasAlignment() method in the set of tilt series. And shouldn't this be enough to distinguish this case?

Comment (azazellochg) [reply novactf/protocols/protocol_tomoReconstruction.py:405]: sure, it make sense to use hasAlignment()

**Comment by Vilax** [comment]
All request have been carried out:

* Check if the tilt series has alignment instead of just removing it
* Remove gold beads params

**Review by azazellochg — APPROVED** [review]

**Pull request merged**